nengo/tests/conftest.py

```diff
@@ -178,6 +189,10 @@ def pytest_addoption(parser):
 
 
 def pytest_runtest_setup(item):
+    if (item.config.getvalue('compare') and
+            not getattr(item.obj, 'compare', None)):
```

Comment:
I would prefer `getattr(item.obj, 'compare', None) is None` here (mostly for readability), unless there's some reason it might be False or some other value that would evaluate as false.

Also, what is this `if` block testing for (i.e. why would we hit it)? Does it just skip the test if it's not marked "compare"? A comment explaining what this is doing would be nice.